pkgs/development/python-modules/icnsutil/default.nix

```
@@ -0,0 +1,30 @@
+{ lib
+, python3
+, fetchFromGitHub
+}:
+
+python3.pkgs.buildPythonPackage rec {
```

pull from python package set
```suggestion
{ lib
, python
, fetchFromGitHub
, buildPythonPackage
}:

buildPythonPackage rec {
```